app/src/main/java/ch/admin/bag/dp3t/networking/FakeWorker.java

```diff
@@ -63,16 +73,26 @@ public FakeWorker(@NonNull Context context, @NonNull WorkerParameters workerPara
 	@NonNull
 	@Override
 	public ListenableWorker.Result doWork() {
-		Logger.d(TAG, "start");
-		DP3T.addWorkerStartedToHistory(getApplicationContext(), "fake");
-		try {
-			executeFakeRequest(getApplicationContext());
-			startFakeWorker(getApplicationContext(), ExistingWorkPolicy.APPEND);
-		} catch (IOException | ResponseError e) {
-			Logger.e(TAG, "failed", e);
-			return Result.retry();
+		long now = System.currentTimeMillis();
+		long t_dummy = getInputData().getLong(KEY_T_DUMMY, now);
+		while (t_dummy < now) {
+			Logger.d(TAG, "start");
+			if (t_dummy >= now - FACTOR_HOUR_MILLIS * MAX_DELAY_HOURS) {
```

Comment:
I would prefer a comment here, similar to the iOS version that explains that we skip fake-uploads that should have been made more than MAX_DELAY_HOURS ago.